test/sbml2symbolics.jl

```diff
@@ -0,0 +1,43 @@
+using Symbolics
+
+# sbmlfile = "reactionsystem_01.xml"
```

Comment:
?

Reply:
I think this is fixed now, but I do not see the "Outdated" label.